1.9/administration/upgrading.md

```diff
@@ -98,6 +98,27 @@ Proceed with upgrading every master node using the following procedure. When you
     - Monitor the Exhibitor UI to confirm that the Master rejoins the ZooKeeper quorum successfully (the status indicator will turn green).  The Exhibitor UI is available at `http://<dcos_master>:8181/`.
     - Verify that `curl http://<dcos_master_private_ip>:5050/metrics/snapshot` has the metric `registrar/log/recovered` with a value of `1`.
     - Verify that `http://<dcos_master>/mesos` indicates that the upgraded master is running Mesos 1.0.1.
+    
+1. Migrate Exhibitor Filesystem Paths
+
+    __Important__: This only applies to clusters that were initially installed with a version of DC/OS < 1.8, if the cluster was installed starting with 1.8 the updated paths are already in use.
```

Comment:
Suggest: "... < 1.8. If the cluster was installed with DC/OS version 1.8 or above, the updated paths are already in use."